pkg/linode-bs/nodeserver_all.go

```diff
@@ -0,0 +1,47 @@
+//go:build !windows
+// +build !windows
```

Comment:
Do we want to support legacy build tags? `go fix` will remove these anyway and I don't think we support a toolchain version that doesn't know about `go:build`

Reply:
default for goimports fixes this i believe... pretty sure i only put in the +build and it added it for me. let me investigate but I agree with you, probably no reason for both at this point